### ISSUE TITLE
The localhost web page doesn't work

### ISSUE THREAD

**Issue by N1kkitta** [Jaggernaut555/midi-mixer-spotify#5]
When i put the URL, a message of error appear, and i can't to see or copy the Access Token and the refresh, do you know if it is about the firewall or something? Or May be the page not working anymore? Thank you for your help i tried to follow de README achive from the top to the bottom, but i have this issus.

**Comment by Jaggernaut555** [comment]
On step 8
> On this plugin's `Settings` page click the `Run Login Server` button

it tries to start a tiny web server on your machine so you can get the spotify tokens. When you click it does the server status change to `Running`?
![image](https://github.com/user-attachments/assets/e5961cd0-1498-48b2-82dc-3673f8836051)

**Comment by N1kkitta** [comment]
No, i follow the steps but nothing happen when i push "Run Login Server" button. Only when i push the "disabled" button on the top right, and the access and refresh automatically fill with some short password of 4 characters. But still "not Running" dont know why.
